packages/sn-controls-react/package.json

```diff
@@ -62,19 +59,11 @@
     "@types/react-dom": "^16.0.7",
     "@types/react-redux": "^7.0.1",
     "@types/react-responsive": "^3.0.2",
-    "autoprefixer": "^9.4.10",
-    "chalk": "2.4.2",
-    "cpx": "^1.5.0",
-    "date-fns": "^2.0.0-alpha.21",
-    "dotenv": "^7.0.0",
-    "fs-extra": "^7.0.1",
-    "object-assign": "4.1.1",
-    "postcss-flexbugs-fixes": "4.1.0",
-    "promise": "^8.0.3",
     "prop-types": "^15.7.2",
     "react": "^16.8.4",
-    "react-dev-utils": "^8.0.0",
-    "react-dom": "^16.8.4"
+    "react-dom": "^16.8.4",
+    "react-redux": "^6.0.1",
+    "redux": "^4.0.1"
```

Comment:
I think we should refactor the *controls-react* package to work without redux in the future